src/apps/vguest/vguest_dev.lua

```diff
@@ -8,6 +8,7 @@ local S         = require('syscall')
 local pci       = require("lib.hardware.pci")
 local bit       = require('bit')
 local gvring    = require('apps.vguest.guest_vring')
+require("lib.checksum")
```

Comment:
It looks like the only thing that requires `lib.checksum` is the `C.prepare_packet()` function. In that case, it looks to me like it is enough to require `lib.checksum_h`.

What caught my attention is that the result from `require("lib.checksum")` is not saved into a variable, which hinted to me that functions from `lib.checksum` are not really used, but importing it would in turn require some other module which defines the used functions — and it seems more correct to require the exact module which defines the needed functions.